### ISSUE TITLE
Add MKT and UR parameters to Bing Map Api

### ISSUE THREAD

**Issue by nabilelmahiri** [openlayers/openlayers#15019]
Hello, at the moment the bing map api take into consideration the culture param only, is it possible to add the mkt (market parameter) and UR ( user region ), hence those are very important too to provide a much suitable experience to the user, i am forced to appended it to the api call each time. ( that would be a great addition :D)

in the ol.js

`const i="https://dev.virtualearth.net/REST/v1/Imagery/Metadata/"+this.imagerySet_+"?uriScheme=https&include=ImageryProviders&key="+this.apiKey_+"&mkt=ma-Ma&countryRegion=Morocco&ur=ma&c="+this.culture_;`

Thanks

**Comment by tschaub** [comment]
Looks like the description of these parameters might be here https://learn.microsoft.com/en-us/bingmaps/rest-services/common-parameters-and-types/

**Comment by mike-000** [comment]
#11814 is related to this.
Another option which might be useful is custom style
https://stackoverflow.com/questions/60849441/how-to-use-own-styling-of-bing-maps-in-openlayers

**Comment by nabilelmahiri** [comment]
@tschaub Yes, those are the map control parameters, more details in this link https://learn.microsoft.com/en-us/bingmaps/v8-web-control/creating-and-hosting-map-controls/setting-map-control-parameters

**Comment by nabilelmahiri** [comment]
@mike-000, nice trick in the link you have provided to append the params to the api key string, much better than changing the openlayer library, i was hopping that we get an official way to benefit from those params instead on relaying on workarounds.

**Comment by mike-000** [comment]
As there are multiple addition parameters available which a user might want to include, instead of attempting to explicitly support each one in the library it might be better to have an single option similar to `dimensions` in WMTS where the user can add additional parameters as required.

**Comment by nabilelmahiri** [comment]
@mike-000, there is not actually much params, its divided in some categories , the most important one is 'User Context Parameters'[and](https://learn.microsoft.com/en-us/bingmaps/rest-services/common-parameters-and-types/user-context-parameters), the others we can add to dimensions as you are suggesting
